scripts/prepdocs.py

```diff
@@ -74,18 +96,74 @@ def remove_blobs(filename):
             if args.verbose: print(f"\tRemoving blob {b}")
             blob_container.delete_blob(b)
 
-def split_text(pages):
+def table_to_html(table):
+    html = "<table>"
+    rows = [sorted([cell for cell in table.cells if cell.row_index == i], key=lambda cell: cell.column_index) for i in range(table.row_count)]
+    for row_cells in rows:
+        html += "<tr>"
+        for cell in row_cells:
+            tag = "th" if (cell.kind == "columnHeader" or cell.kind == "rowHeader") else "td"
+            cell_spans = ""
+            if cell.column_span > 1: cell_spans += f" colSpan={cell.column_span}"
+            if cell.row_span > 1: cell_spans += f" rowSpan={cell.row_span}"
+            html += f"<{tag}{cell_spans}>{cell.content}</{tag}>"
```

Comment:
Is cell.content already HTML-escaped? Do we want it to be? On one hand there might be valid cases where GPT can generate additional markup within cells, on the other hand unescaped strings could have "<" ">" or other characters that will make it invalid HTML. The FE already sanitizes it for security, but there's still the issue of breaking the format. A simple solution would be to just do html.escape() on this, a more complicated one would be to see what are well-formed tags and what's just accidental characters that need to be escaped. I would propose that we just do html.escape() for now.